corehq/apps/reports/views.py

```diff
@@ -1856,6 +1856,7 @@ class TableauServerView(BaseProjectReportSectionView):
     page_title = gettext_lazy('Tableau Server Config')
     template_name = 'hqwebapp/crispy/single_crispy_form.html'
 
+    @method_decorator(require_superuser)
```

Comment:
Not really part of this PR, but FWIW you can pass a `name` argument to `method_decorator` and stick it on the class, so you don't need to override a method just to decorate it:
```python
@method_decorator(require_superuser, name="dispatch")
class TableauServerView(BaseProjectReportSectionView):
    ...
```